contribs/dvrp/src/test/java/org/matsim/contrib/dvrp/passenger/PassengerEngineTestFixture.java

```diff
@@ -107,7 +107,10 @@ void addPersonWithLeg(Link fromLink, Link toLink, double departureTime) {
 	void assertPassengerEvents(Event... events) {
 		assertThat(recordedEvents.size()).isGreaterThanOrEqualTo(events.length);
 		var recordedPassengerEvents = recordedEvents.stream()
-				.filter(e -> e instanceof HasPersonId && ((HasPersonId)e).getPersonId().equals(PERSON_ID));
+				.filter(e ->
+						e instanceof HasPersonId && ((HasPersonId)e).getPersonId().equals(PERSON_ID) ||
+								e instanceof AbstractPassengerRequestEvent
```

Comment:
Why is this change made?

Reply:
@michalmac because the abstractpassengerrequest events do not implement the HasPersonid anymore, as they do not point to a single person any longer